pkg/serverless/trigger/events/events.go

```diff
@@ -334,3 +334,46 @@ type LambdaFunctionURLRequestContextHTTPDescription struct {
 	SourceIP  string
 	UserAgent string
 }
+
+// StepFunctionEvent is the event you get when you put
+// ```
+//
+//	"Parameters": {
+//	       "Payload.$": "States.JsonMerge($$, $, false)",
+//	       "FunctionName": "${lambdaArn}"
+//	     },
+//
+// ```
+// into a Lambda:Invoke task state, as is recommended by https://docs.datadoghq.com/serverless/step_functions/installation?tab=custom
+// This isn't an "official" event, as a default StepFunction invocation will just contain {}
+type StepFunctionEvent struct {
+	Payload StepFunctionPayload
+}
+type StepFunctionPayload struct {
+	Execution    StepFunctionExecution
+	StateMachine StepFunctionStateMachine
+	State        StepFunctionState
+}
+
+// StepFunctionExecution is the JSON object that represents a StepFunction execution, that is passed as part of the Lambda function input
+type StepFunctionExecution struct {
+	ID           string
+	StartTime    string
+	Name         string
+	RoleArn      string
+	RedriveCount int64
+	Input        interface{}
+}
+
+// StepFunctionStateMachine is the JSON object that represents a State Machine
+type StepFunctionStateMachine struct {
+	ID   string
+	Name string
+}
+
+// StepFunctionState is the JSON object that represents the state that invoked a Lambda execution
```

Comment:
Can we update these doc comments to match the others in this file?

Reply:
Okay, now I'm confused. I'm not seeing any Step Function event types defined [by `aws/aws-lambda-go/events`](https://pkg.go.dev/github.com/aws/aws-lambda-go@v1.47.0/events]. Where are these all coming from then?